front/pages/w/[wId]/workspace/index.tsx

```diff
@@ -206,12 +205,14 @@ export default function WorkspaceAdmin({
           <Page.Vertical align="stretch" gap="md">
             <Page.H variant="h4">Analytics</Page.H>
             <Page.Horizontal gap="lg">
-              <QuickInsights owner={owner} />
-              <ActivityReport
-                isDownloading={isDownloadingData}
-                monthOptions={monthOptions}
-                handleDownload={handleDownload}
-              />
+              <div className="grid w-full grid-cols-2 gap-4">
```

Comment:
This will break the smaller screen view? 

If I understand it correctly Page.Horizontal have flex for smaller screen:
<img width="447" alt="Screenshot 2025-04-07 at 12 15 56" src="https://github.com/user-attachments/assets/8464a9d0-658b-47a2-9851-2c9dd7af6199" />

before:
<img width="504" alt="Screenshot 2025-04-07 at 12 06 24" src="https://github.com/user-attachments/assets/a241c252-3eb1-488f-a6c8-675cb09b7ae1" />

now:
<img width="495" alt="Screenshot 2025-04-07 at 12 05 28" src="https://github.com/user-attachments/assets/a1dc6bb8-bd4d-47ea-b854-45278dcdf7d9" />

Reply:
For some reason I thought the activity report was hidden on mobile.. 🙈